src/main/java/org/folio/ed/handler/FeedbackChannelHandler.java

```diff
@@ -30,6 +30,10 @@ public Object handle(String payload, Configuration configuration) {
       var errorCode = extractErrorCode(payload);
       switch (errorCode) {
         case INVENTORY_IS_AVAILABLE:
+//          return buildPickRequestMessage(remoteStorageService.getRetrievalQueueRecords(
```

Comment:
@siarhei-charniak  - can you please suggest if I can call getRetrievalQueueRecords each time for this case rather than using the getRetrievalByBarcode method which contains a retrieval map (but its not updated).

Reply:
@gurleenkaurbp, retrieval map is updated on each poll, so I think you may use getRetrievalByBarcode method